diff --git a/dist/runtime/web/color-scheme.js b/dist/runtime/web/color-scheme.js
index c8fb1d2d9eb59204349cf80aeb56cbc281e81771..6f8494097e57b1a11745db3a02ed7fb839960409 100644
--- a/dist/runtime/web/color-scheme.js
+++ b/dist/runtime/web/color-scheme.js
@@ -33,6 +33,8 @@ else if ("window" in globalThis) {
         const flags = darkModeFlag.split(" ");
         darkMode = flags[0];
         darkModeValue = flags[1];
+        if (darkMode === "media")
+            return;
         exports.colorScheme.set(globalThis.window.document.documentElement.classList.contains(darkModeValue)
             ? "dark"
             : "system");
